docs/docs/quick-start.md

```diff
@@ -1,66 +1,66 @@
 ---
-title: Quick Start
+शीर्षक: क्विक स्टार्ट
 ---
 
-This quick start is intended for intermediate to advanced developers. For a gentler intro to Gatsby, [head to our tutorial](/tutorial/)!
+यह क्विक स्टार्ट मध्यवर्ती से उन्नत डेवलपर्स के लिये है| गैट्सबी के एक विनम्र परिचय के लिए, [हमारे ट्यूटोरियल देखें](/tutorial/)!
 
-## Use the Gatsby CLI
+## गैट्सबी सीएलआई का उपयोग
```

Comment:
`गैट्सबी` => `Gatsby`

We are not translating `Gatsby` word, Can you find and replace this in whole file